commonutils/src/test/java/io/intrepid/commonutils/util/MockedTextUtils.java

```diff
@@ -0,0 +1,28 @@
+package io.intrepid.commonutils.util;
+
+import android.text.TextUtils;
+
+import org.mockito.Mockito;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+import org.powermock.api.mockito.PowerMockito;
+
+import static org.mockito.Matchers.any;
+
+/**
+ * Mocks the {@link TextUtils} class.
+ *
+ * Created by aspaans on 5/6/16.
+ */
+public class MockedTextUtils {
+    public static void setup() {
+        PowerMockito.mockStatic(TextUtils.class);
+        Mockito.when(TextUtils.isEmpty(any(CharSequence.class))).then(new Answer<Boolean>() {
+            @Override
```

Comment:
let's just make isEmpty a method in our StringUtils class. We should move away from using TextUtils.isEmpty() since it's a pain to get it to work in unit tests